docs/blog/es-tooling.md

```diff
@@ -10,43 +10,43 @@ head:
     - property: og:url
       content: https://e18e.dev/blog/es-tooling
   - - meta
     - property: og:description
-      content: ES Tooling - a community lead, e18e-focused collection of tools for the ecosystem
+      content: ES Tooling, a community-led, e18e-focused collection of tools for the ecosystem
 ---
 
 # ES Tooling
 
 _July 22, 2024_
 
-The e18e initiative has seen huge growth since launch. We've seen hundreds of performance contributions upstream to popular projects, and the creation of many community lead tools.
+The e18e initiative has seen huge growth since launch. We've seen hundreds of performance contributions upstream to popular projects, and the creation of many community-led tools.
 
 A notable example of this is the [ES Tooling](https://github.com/orgs/es-tooling/) community GitHub organisation.
 
 If you haven't yet encountered e18e, read more in the [guide](https://e18e.dev/guide/#why-e18e) and [join the community](https://chat.e18e.dev/).
 
 ## A home for e18e-focused tools
 
-Over the last few months, many of us have collaborated and connected through the e18e community. The ES tooling organisation is a result of that - a place for us to store and own forks, tools, and other code bases we're working on together.
+Over the last few months, many of us have collaborated and connected through the e18e community. The ES tooling organisation is a result of that: a place for us to store and own forks, tools, and other codebases we're working on together.
 
-A few popular projects that were running solo until now, have come together to create this:
+A few popular projects that were running solo until now have come together to create this:
 
 - [ecosystem-cleanup](https://github.com/43081j/ecosystem-cleanup).
 - [module-replacements](https://github.com/es-tooling/module-replacements).
 - [module-replacements-codemods](https://github.com/thepassle/module-replacements-codemods)
 - [fd-package-json](https://github.com/es-tooling/fd-package-json)
 
-While e18e exists to connect us all and provide a space to collaborate, it doesn't exist to own the projects coming out of the initiative. That is where projects like es-tooling, unjs and more come in to play.
+While e18e exists to connect us all and provide a space to collaborate, it doesn't exist to own the projects coming out of the initiative. That is where projects like es-tooling, unjs and more come into play.
 
-**We expect es-tooling will be one of many community lead orgs with an e18e focus.**
+**We expect es-tooling will be one of many community-led organizations with an e18e focus.**
 
 ## Ecosystem cleanup
 
-One of the projects which kicked much of this initiative off - the ecosystem cleanup project has now moved into the ES Tooling organisation to allow better visibility and maintenance.
+One of the projects which kicked much of this initiative off, the ecosystem cleanup project, has now moved into the ES Tooling organization to allow better visibility and maintenance.
```

Comment:
organisation is correct spelling in British English. I don't have much of a preference, but will note that this page is now inconsistent in how the word is spelled

Reply:
Right, I tried to make this consistent, but now I see there are other instances. If you let me know which variant of English you'd like to use (American or British), I'll make the required changes.

Reply:
American english